server/ldap_check.rb

```diff
@@ -43,27 +48,45 @@ def initialize(args={})
       @conf_values[k] = args[k]
     }
 
+    configuration['cache_seconds'] = @default_cache_seconds unless (configuration['cache_seconds'])
+
     if logger.debug?
       logger.debug "final conf values"
-      @conf_values.each_key { |k| logger.debug "key: [#{k}] value: [#{@conf_values[k]}]" }
+      configuration.each_key { |k| logger.debug "key: [#{k}] value: [#{configuration[k]}]" }
     end
 
     ## save members in this group if one is specified.
-    save_group_members args['group'] if args['group']
+    #save_group_members args['group'] if args['group']
 
   end
 
 
-  # this provides access to verify / debug configuration.
+  # Provide access to configuration hash
   def configuration
     @conf_values
   end
 
+  # provide access to count of LDAP update calls
+  def updateCount
```

Comment:
It used to document the number of updates.  Since it is only used in this class I removed the access method and will use direct access.